### PR TITLE
Epic CSS Overhaul

### PR DESCRIPTION
**Pull request by beatrixbetsycarroll**
@matthewhuff89 @dm-istomin

### CODE REVIEW

File: spec/features/sessions_spec.rb [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+# #     expect(page).to have_content user.username
+# #   end
 
-  scenario "a user tries to log in with email and see their personal user page" do
-    user = create(:user)
-    visit login_path
-    fill_in "session_user_name", with: user.email
-    fill_in "session_user_password", with: user.password
-    click_button "Log In"
+# #   scenario "a user tries to log in with email and see their personal user page" do
```
Comment (sgharms) [inline spec/features/sessions_spec.rb]: tumblr_mgkokoikad1s0xrbio1_250

It's doubly commented out so it's double invisible.

File: spec/features/sessions_spec.rb [hunk 2]
```diff
@@ -1,28 +1,28 @@
-require "rails_helper"
+# require "rails_helper"
```
Comment (sgharms) [inline spec/features/sessions_spec.rb]: A full file of commented out tests?  :(

File: spec/features/decks_spec.rb [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
-    expect(page).to have_content '+'
-    expect {
-      click_link '+'
-    }.to change(deck.cards, :count).by(1)
-    expect {
-      click_link '-'
-    }.to change(deck.cards, :count).by(-1)
-  end
-end
+#     visit deck_path(deck)
```
Comment (sgharms) [inline spec/features/decks_spec.rb]: :(

File: app/views/pages/splash.html.erb [hunk 4]
```diff
@@ -1,7 +1,14 @@
-<div id="splash_pic">
-  <center>
+<div id="splash_pic" class = "u-max-full-width class">
+	<%= image_tag("brokenclock.png", class: "u-max-full-width clock") %>
```
Comment (sgharms) [inline app/views/pages/splash.html.erb]: indentation is nuts here.  why 6 spaces?

File: app/views/games/show.html.erb [hunk 5]
```diff
@@ -1,8 +1,8 @@
-
-
 <div id="game-container">
+
 <div id="opponent-move"></div>
 <div id="player-move"></div>
 <%= render partial: 'player_1_scoreboard' %>
+
```
Comment (sgharms) [inline app/views/games/show.html.erb]: a commit to add a whitespace line?  No.

File: app/views/decks/show.html.erb [hunk 6]
```diff
@@ -1,43 +1,39 @@
...
       <ol>
         <% @cards.each do |card| %>
-          <li>
-          <%= link_to "+", add_card_path(card_id: card), method:"POST" %>
-            <div class="mini_card">
-              <%= link_to (image_tag card.picture_url, class: "portrait", title: card.name), card_path(card) %>
-            </div>
-          </li>
+            
+            <%= render partial: 'add_card', locals: { :@card => card} %>
```
Comment (sgharms) [inline app/views/decks/show.html.erb]: Looks inconsistent.

**Comment by sgharms** [comment]
@dm-istomin, @beatrixbetsycarroll , @matthewhuff89 Why are we merging this in at 1:25 am on a day that was supposed to end 85 minutes ago and whose theme was "minor refactorings, no new features?"

In a professional context it would be feasible for me (as your tech lead) to find the last commit that you have which was by the rules, reset your origin branch to that, and then lock the repo so that no further commits could come in.

**Comment by kaylorburdell** [comment]
dude go to bed

**Comment by dm-istomin** [comment]
@sgharms Sorry Steven, we had a pretty horrible JS bug that took us ~6 hours to resolve. We're not adding new features, just cleaning up the styles for the presentation; should be done soon!

**Comment by sgharms** [comment]
@kaylorburdell 

haters

**Pull request merged**